templates/admission-policy.yaml

```diff
@@ -0,0 +1,56 @@
+{{- $kubeVersion := semverCompare ">1.30.0" .Values.global.clusterConfiguration.kubernetesVersion }}
```

Comment:
"kubeVersion" name has too wide meaning and no indication that it is actually a boolean flag.